src/main/webapp/app/admin/metrics/blocks/metrics-modal-threads/metrics-modal-threads.component.ts

```diff
@@ -9,8 +9,21 @@ import { Thread, ThreadState } from '../../metrics.model';
 })
 export class MetricsModalThreadsComponent implements OnInit {
     ThreadState = ThreadState;
-    threadStateFilter?: string;
+
+    private _selectedThreadState?: ThreadState;
```

Comment:
There is a guideline that says, `Do not use “_” as a prefix for private properties` ; I am not sure if it still valid, but I guess so... do you still want to keep the naming?